packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx

```diff
@@ -7,29 +7,46 @@ export interface IButtonProps {
     className?: string,
     disabled?: boolean,
     caption: string,
+    color?: string
```

Comment:
color is mainly used for font colors in css.  just double checking here for what is this used? i only see it used for a border color

Reply:
i guess this comes from the secondary button. there the border and font color was always meant to be the same. so that's why color is also used for the border over there.

Reply:
Yes I made them equal